### DISCUSSION TITLE
After Migrating Supabase at app.supabase.com now Storage is not working

### DISCUSSION THREAD

**Discussion by iampopal** [supabase/supabase#29700] [Questions]
After migrating storage are failing and working any work around or how to fix the storage issue.

**Comment by GaryAustin1** [comment]
Can you explain what you did?  Not clear what migrating storage means with no context.  Did you use a guide?

**Comment by iampopal** [comment]
I had received an email to update your project to latest version of
Postgresql and I followed up that link and updated the project to latest
version. Now the issue is that storage and storage api is not working. The
issue is shown in the following image too.

Thank you in advance.

**Reply by GaryAustin1** [reply]
I've also commented on your Discord one.
Contacting support is what you need to do after doing all the things discussed there.

**Comment by iampopal** [comment]
Thank you for commenting. We have a a huge user base. Seeing this storage
down bring the trust down. The support is not responding. Can you help give
me a person contact who can rapidly answer me.
Thank you.
